### ISSUE TITLE
Fetching data page missing in the docs

### ISSUE THREAD

**Issue by shrubway** [solidjs/solid#2232]
### Describe the bug

https://docs.solidjs.com/guides/fetching-data leades to 404. 
This is a crucial info and it's shame that is missing from the docs. 
Please fix it. 

### Your Example Website or App

https://docs.solidjs.com/guides/fetching-data

### Steps to Reproduce the Bug or Issue

Go to https://docs.solidjs.com/guides/fetching-data

### Expected behavior

Showing fetching data info. 

### Platform

- Browser: All

**Comment by dhairya-cse** [comment]
[shrubway](https://github.com/shrubway) It's not showing 404 for me, could you check again?

**Comment by ryansolid** [comment]
Yeah I wasn't able to reproduce yesterday or today. It may have been some some odd caching/deployment timing thing.
